# Generated for NaoQi method ALAudioDevice.getLeftMicEnergy
# Returns the energy of the left microphone signal

---
# energy of the left microphone signal
float32 value


